client/rollup.config.js

```diff
@@ -0,0 +1,72 @@
+import vue from 'rollup-plugin-vue';
+// use this instead of the official @rollup/plugin-typescript
+// because official doesn't work and this one actually has
+// more community support
+import typescript from 'rollup-plugin-typescript2';
+import scss from 'rollup-plugin-scss';
+import cleaner from 'rollup-plugin-cleaner';
+
+const clean = true;
+const tsconfig = 'tsconfig.rollup.json';
+const external = [
+  '@vue/composition-api',
+  '@flatten-js/interval-tree',
+  'd3',
+  'geojs',
+  'lodash',
+  'vue',
+];
```

Comment:
Is this ultimately specified to reduce bundle size?

Reply:
It does that, but only because there's just no other option.  Otherwise you run into the issue where multiple instances of Vue (and every other dependency) are instantiated and nothing works.

The point is to defer dependency resolution to the consumer.